integration_files/E2_Series/common_files/directed_tests/asm/hello_world.s.map

```diff
@@ -0,0 +1,219 @@
+Archive member included to satisfy reference by file (symbol)
+
```

Comment:
@saqibsaeed-lm This is an autogenerated file which I think is just an output of something and is not required for anything. Can you confirm this? If it is so then please remove this file.

Reply:
removed this file after discussion with najeeb